runtests.sh

```diff
@@ -36,7 +36,7 @@ run_static_tests() {
     python3 -m flake8 .
 
     echo "Running mypy"
-    mypy --python-version 3.6 -p snapcraft
+    mypy --python-version 3.6 snapcraft tests
```

Comment:
you might be able to drop `--python-version 3.6`, I missed the change on mypy.ini (couldn't find where it was back then)